flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -187,8 +187,6 @@
 
 	private final ResultPartition[] producedPartitions;
 
-	private final ResultPartitionWriter[] writers;
```

Comment:
You have failing checkstyle with unused `ResultPartitionWriter` import in this file.